lib/lbt/bundle/Builder.js

```diff
@@ -411,7 +411,8 @@ class BundleBuilder {
 				try {
 					fileContent = JSON.stringify( JSON.parse( fileContent) );
 				} catch (e) {
-					log.error(e);
+					log.verbose("error while parsing JSON file %s", module);
```

Comment:
Maybe add a hint why the file is parsed and that the file is just bundled as is.